src/Admin/AdminExtensionInterface.php

```diff
@@ -126,42 +134,42 @@ public function getPersistentParameters(AdminInterface $admin);
     /**
      * @param object $object
      *
-     * @return void
```

Comment:
I saw you removed lot of `@return void`.

For phpstan `@return mixed` is the default behavior. So `@return void` is needed to have a better analysis.

Reply:
Are you sure? Do you have a link where this is explained?

Reply:
https://github.com/phpstan/phpstan/issues/2842#issuecomment-573139424